builder/dockerfile/dispatchers.go

```diff
@@ -205,7 +205,8 @@ func dispatchTriggeredOnBuild(d dispatchRequest, triggers []string) error {
 		}
 		cmd, err := instructions.ParseCommand(ast.AST.Children[0])
 		if err != nil {
-			if instructions.IsUnknownInstruction(err) {
```

Comment:
Looks like this was removed in the vendored version; should this be in the same commit to not break git bisect?

Reply:
I guess this is what `git bisect skip` is for. I can put everything to a single commit if you like. Though this is easier to review.

Reply:
With github hiding vendored files, it's probably ok to squash the commits.